src/models/disassemblyoutput.cpp

```diff
@@ -308,22 +308,26 @@ DisassemblyOutput DisassemblyOutput::disassemble(const QString& objdump, const Q
     }
     arguments.append(binary);
 
-    asmProcess.start(objdump, arguments);
+    asmProcess.start(processPath, arguments);
 
     if (!asmProcess.waitForStarted()) {
-        disassemblyOutput.errorMessage += QApplication::translate("DisassemblyOutput", "Process was not started.");
+        disassemblyOutput.errorMessage += QApplication::translate("DisassemblyOutput", "Process was not started.")
+            + QStringLiteral("\n")
+            + QApplication::tr("command: %1 %2").arg(processPath, arguments.join(QLatin1Char(' ')));
         return disassemblyOutput;
     }
 
     if (!asmProcess.waitForFinished()) {
         disassemblyOutput.errorMessage +=
-            QApplication::translate("DisassemblyOutput", "Process was not finished. Stopped by timeout");
+            QApplication::translate("DisassemblyOutput", "Process was not finished. Stopped by timeout")
```

Comment:
dito

Reply:
will fix the string puzzles, I likely was concerned about not breaking the existing translations